default/python/universe_generation/util.py

```diff
@@ -43,5 +45,12 @@ def report_error(msg):
     print(msg, file=sys.stderr)
 
 
+def iterate_pairs(lst: Iterable[int], lst2: Iterable[int]) -> Iterable[Tuple[int, int]]:
```

Comment:
It is not Iterable it is Iterator.  It can be iterated only once.

```
a = filter(None, 'abc')
list(a)
['a', 'b', 'c']
list(a)
[]
```

Reply:
Fixed returning type.